src/dialogs/settingsdialog.h

```diff
@@ -48,6 +48,7 @@ Q_OBJECT
         ToolbarPage,
         DebugOptionPage,
         EditorPage,
+        PanelsPage,
```

Comment:
the enum is autoincrementing from one constant to the next and the number has to be corresponding to the "whatsThis" number, the position you took was already taken